src/pdfviewer/PDFDocument.cpp

```diff
@@ -2098,10 +2108,14 @@ void PDFWidget::setPageOffset(int offset, bool setAsDefault, bool refresh){
 		pageOffset = offset;
 	else {
 		pageOffset = gridx - 1;
-		globalConfig->pageOffset = pageOffset;
 	}
-	if (!setAsDefault)
-		globalConfig->pageOffset = pageOffset;
+	if (!setAsDefault) {
```

Comment:
the setAsDefault was the way of distinguishing embedded/windowed
Having now an additional "embedded" makes a mess of the concept.

Reply:
no. as explained above there are cases where `pageOffset` needs to be changed temporarily, i.e. without updating the profile. That's what `setAsDefault` is used.